gips/utils.py

```diff
@@ -78,7 +79,7 @@ def verbose_out(obj, level=1, stream=sys.stdout):
         for o in obj:
             print(o, file=stream)
 
-VerboseOut = verbose_out # TODO deprecate then remove
+VerboseOut = verbose_out # VerboseOut name is deprecated
```

Comment:
My thought is that this should be wrapped, such that it announces deprecation at an appropriate verbose-level to aid the transition.  Possibly adding a `@deprecated` decorator to gips.